pkgs/by-name/ms/msedgedriver/package.nix

```
@@ -0,0 +1,55 @@
+{
+  autoPatchelfHook,
+  fetchurl,
+  glib,
+  lib,
+  nspr,
+  nss,
+  stdenv,
+  unzip,
+  xorg,
+}:
+let
+  channel = "stable";
+  version = "129.0.2792.91";
+  hash = "sha256-69+quFNgjioVBtsRSZDPBYXWhtCvX6cZ5TqGhhf5Ht8=";
+in
+stdenv.mkDerivation {
+  pname = "msedgedriver";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://msedgedriver.azureedge.net/${version}/edgedriver_linux64.zip";
+    inherit hash;
+  };
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+    glib
+    xorg.libxcb
+    nspr
+    nss
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    unzip
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    unzip $src
+    install -m777 -D "msedgedriver" $out/bin/msedgedriver
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "";
+    description = "";
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
+    license = lib.licenses.unfree;
+    maintainers = with lib.maintainers; [ cholli ];
+    platforms = lib.platforms.linux;
+    mainProgram = "msedgedriver";
+  };
+}
```

Closing parenthesis for `finalAttrs`
```suggestion
})
```